Lib/inspect.py

```diff
@@ -511,13 +519,17 @@ def iscode(object):
                             | 256=iterable_coroutine | 512=async_generator
         co_freevars         tuple of names of free variables
         co_posonlyargcount  number of positional only arguments
+        co_positions        returns an iterator of source code positions for each bytecode instruction
         co_kwonlyargcount   number of keyword only arguments (not including ** arg)
         co_lnotab           encoded mapping of line numbers to bytecode indices
+        co_lines            returns an iterator that yields successive bytecode ranges
         co_name             name with which this code object was defined
         co_names            tuple of names other than arguments and function locals
         co_nlocals          number of local variables
         co_stacksize        virtual machine stack space required
-        co_varnames         tuple of names of arguments and local variables"""
+        co_varnames         tuple of names of arguments and local variables
+        co_qualname         fully qualified function name
+        replace             return a copy of the code object with new new values"""
```

Comment:
How about grouping `co_lines` and `replace` as methods? they need to be invoked, whereas the others are more like properties.

Reply:
Or add `()` to methods.